From 38aebbca64e68c729d07b09bf62af405e87db144 Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Fri, 7 Feb 2025 19:12:04 +0100
Subject: [PATCH] [client,common] Avoid use of reserved types

__off_t is a internal type of glibc and does not exist on other libc's.
---
 client/common/client_cliprdr_file.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/common/client_cliprdr_file.c b/client/common/client_cliprdr_file.c
index f1f220e528a4..e895f8b5684c 100644
--- a/client/common/client_cliprdr_file.c
+++ b/client/common/client_cliprdr_file.c
@@ -775,7 +775,7 @@ static void write_file_attributes(CliprdrFuseFile* fuse_file, struct stat* attr)
 	{
 		attr->st_mode = S_IFREG | (fuse_file->is_readonly ? 0444 : 0644);
 		attr->st_nlink = 1;
-		attr->st_size = WINPR_ASSERTING_INT_CAST(__off_t, fuse_file->size);
+		attr->st_size = WINPR_ASSERTING_INT_CAST(off_t, fuse_file->size);
 	}
 	attr->st_uid = getuid();
 	attr->st_gid = getgid();
